### PR TITLE
Lolyparty

### LINKED ISSUE #909: [NG] Migrate MLH2020 page to layer5-ng
**Current Behavior**
The current website has MLH2020 page over [here.](https://layer5.io/programs/mlhfellowship/2020)

**Desired Situation**
Migrate the MLH2020 page to layer5-ng. Please look into how various [programs](https://github.com/layer5io/layer5/tree/layer5-ng/src/collections/programs) have been migrated for reference.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by lolyparty**
**Description**

This PR fixes #909 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/programs/mlh-2020/index.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import { ProgramsWrapper } from "../Programs.style.js";
+import Button from "../../../reusecore/Button";
+import mlh_logo from "./mlh.svg";
+
+<ProgramsWrapper>
+
+<img src={mlh_logo}/>
+<h2><b>MLH Fellowship Program</b></h2>
+<div>
+The Major League Hacking’s <a href="https://https//fellowship.mlh.io/">MLH Fellowship</a> is an internship alternative for software engineers. Instead of working on a project for just one company, you’ll contribute to Open Source projects that are used by companies around the world. You’ll collaborate with a group of 10 students under the guidance of a professional mentor whose only job is to help you successfully contribute.<br/>
```
Comment (Jashpatel1) [inline src/collections/programs/mlh-2020/index.mdx:17]: Fix this link please

File: src/collections/programs/mlh-2020/index.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+<br/>
+Layer5’s <a href="https://https//meshery.io/">Meshery Project</a> is participating in the MLH Fellowship’s internship program in 2020.
```
Comment (Jashpatel1) [inline src/collections/programs/mlh-2020/index.mdx:20]: Fix this link please

Comment (lolyparty) [reply src/collections/programs/mlh-2020/index.mdx:20]: wow, my bad. I missed that, would fix it asap

Comment (lolyparty) [reply src/collections/programs/mlh-2020/index.mdx]: Done!

**Review by Jashpatel1 — APPROVED** [review]
Nice work here. Thanks @lolyparty

**Comment by Nikhil-Ladha** [comment]
Its better to give more sensible title to the PR so, that it becomes easier for the reviewers to go through the changes.

**Comment by Nikhil-Ladha** [comment]
Link to check this out?

**Comment by Jashpatel1** [comment]
> Link to check this out?
> Or is it yet to be configured?

https://deploy-preview-923--layer5ng.netlify.app/programs/mlh/2020

**Comment by lolyparty** [comment]
> Its better to give more sensible title to the PR so, that it becomes easier for the reviewers to go through the changes.

What do you mean by a sensible PR title? Or you mean commit message?

**Comment by Nikhil-Ladha** [comment]
> > Its better to give more sensible title to the PR so, that it becomes easier for the reviewers to go through the changes.
> 
> What do you mean by a sensible PR title? Or you mean commit message?

Yes, the commit message/subject.

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good.

**Pull request merged**